src/composables/sdk13.ts

```diff
@@ -148,6 +150,31 @@ export function useSdk13({ store }: IDefaultComposableOptions) {
     return sdk;
   }
 
+  /**
+   * drySdk is the sdk instance with no accounts attached.
+   * To use for multisig operations.
+   */
+  async function getDrySdk(): Promise<AeSdk> {
+    if (!drySdk) {
+      const nodeInstance = await createNodeInstance(activeNetwork.value.url);
+      drySdk = new AeSdk({
+        nodes: [{
+          name: activeNetwork.value.name,
+          instance: nodeInstance!,
+        }],
+        onCompiler: new CompilerHttp(activeNetwork.value.compilerUrl),
+      });
+      return drySdk;
+    }
+    const networkId = await sdk.api.getNetworkId();
+    if (activeNetwork.value.networkId !== networkId) {
+      drySdk.pool.delete(sdkCurrentNetwork.name);
+      const nodeInstance = await createNodeInstance(activeNetwork.value.url);
+      drySdk.addNode(activeNetwork.value.name, nodeInstance!, true);
```

Comment:
This optimization is redundant. We can always assume networks are different and proceed. Both approaches make one API request to the node. but in the approach I suggested we can eliminate branching(if-statement). Eliminating branching is a great runtime optimization.

Reply:
irrelevant to what I commented. My suggestion is 
```suggestion
      drySdk.pool.delete(sdkCurrentNetwork.name);
      const nodeInstance = await createNodeInstance(activeNetwork.value.url);
      drySdk.addNode(activeNetwork.value.name, nodeInstance!, true);
```
This works, the optimization using `if` branching is not really an optimization.

Reply:
Got it.

Reply:
As I see, with no network switching `sdk.api.getNetworkId` wouldn't make an extra request because https://github.com/aeternity/aepp-sdk-js/blob/75097952ee689c591973048b0c52ea1c516dcfb5/src/Node.ts#L155-L160
but `createNodeInstance` would do it for sure. So, `if` makes sense.

Reply:
> Eliminating branching is a great runtime optimization

This makes sense only as improving readability. Don't worry about sync code optimizations unless you have warnings like `[Violation] Handler took 83ms of runtime`.